src/style.css

```
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf),
+    url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color-defult: #3d4e61;
+  --color-box__shadow: #5c5c5c5b;
+}
+
+body {
+  font-family: "Avenir", serif;
+  font-weight: 300;
+}
+
+.search {
+  border: none;
+  background: none;
+  outline: none;
+  padding: 0;
+  width: 100%;
+  font-size: 16px;
+  font-family: inherit;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.search::placeholder {
+  color: #000;
+}
+
+.search--form {
+  display: flex;
+  justify-content: center;
+  border: 1px solid #e1e7ed;
+  border-radius: 5px;
+  margin-top: 20px;
+}
+
+.search--form:focus-within {
+  box-shadow: 0 5px 10px var(--color-box__shadow);
+  color: #3d4e61;
+  font-weight: 700;
+}
+
+.search--small {
+  padding-left: 20px;
+  background-position: left;
+  background-size: 11px;
+  font-size: 14px;
+}
+
+.search--big {
+  padding-left: 36px;
+  background-position: left;
+}
+
+.search:focus {
+  color: #3d4e61;
+  font-weight: 700;
+}
+
+.search--form:hover {
+  box-shadow: 0 5px 10px var(--color-box__shadow);
+}
+
+.search--form__small {
+  padding: 12px 13px 11px;
+}
+
+.search--form__big {
+  padding: 25px 26px 26px;
```

```suggestion
  padding: 25px 26px 23px;
```
or 23 for top and bottom cause at the moment your form height is 75 but it should be 70 in the end, test it in dev tools